assets/src/edit-story/components/panels/layer/layer.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { rgba } from 'polished';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { LAYER_HEIGHT } from './constants';
+import LayerContent from './layerContent';
+
+const LayerButton = styled.button.attrs( { type: 'button' } )`
+	display: flex;
+	border: 0;
+	padding: 0;
+	background: transparent;
+	height: ${ LAYER_HEIGHT }px;
+	width: 100%;
+	align-items: center;
+
+	${ ( { isSelected, theme } ) => isSelected && `
+		background: ${ rgba( theme.colors.action, 0.14 ) };
+	` }
+
+	&:focus,
```

Comment:
Remove `&:focus` since we're switching to focusable-only-by-keyboard thing-y. /cc @wassgha

Reply:
I think we're going to have a different way of displaying focus here. But that'll have to wait until UX.